### DISCUSSION TITLE
Link to annotation set in RWPM

### DISCUSSION THREAD

**Discussion by qnga** [readium/annotations#15] [General]
> The JSON file holding an AnnotationSet MUST be represented as a link object in the links collection, with an annotations relation.
Sample 13: A Readium Web Publication Manifest containing a link to an annotations file.

We had trouble interpreting the RWPM spec last year and decided that every resource contained in a package should be in `resources` while `links` was reserved to  resources _external_ to the publication . We found that distinction between _internal_ and _external_ resources useful in implementations. The initial question was: "where is the cover of a Readium package supposed to be?". And the answer was "in `resources`, not `links`".
As it is not obvious, we should discuss the point again if someone disagrees.

**Comment by qnga** [comment]
Actually, the annotation file should probably be considered as _external_ to the publication though _internal_ to the package. It's a bit different from the cover that is internal to both.  In non-package publications, the link could optionally be in `links` for the sake of discoverability (I'm not sure, if it's outside the publication, there might be no real reason), but in packaged publications I'm a bit stuck. Not referencing it, as with EPUBs, could be the approach making the most sense.

**Reply by llemeurfr** [reply]
I'm more comfortable with annotations being considered _external_ to the publication, as they are not provided by the author/publisher (in general).  

Having a different way to link annotations in unpackaged and packaged publications would be illogical (and complex at implementation time). I prefer having this link to annotations in packaged publications also.
